app/models/region/RegionTable.scala

```diff
@@ -152,6 +154,36 @@ object RegionTable {
   }
 
   /**
+    * Get a named easy region that is amongst the least audited regions
+    * Used for anonymous users
+    *
+    * @return
+    */
+  def selectALeastAuditedEasyRegion: Option[NamedRegion] = db.withSession { implicit session =>
+
+    // Assign one of the least-audited regions that are easy.
+    val completions: List[RegionCompletion] =
+      RegionCompletionTable.regionCompletions
+        .filterNot(_.regionId inSet UserCurrentRegionTable.difficultRegionIds)
+        .filter(region => region.auditedDistance / region.totalDistance < 1.0)
+        .sortBy(region => region.auditedDistance / region.totalDistance).take(10).list
+
+    val regionId: Int = completions match {
+      case Nil =>
+        // Indicates amongst the unaudited regions of the user, there are no unaudited regions across all users
+        // In this case, pick any easy region amongst regions that are not audited by the user
```

Comment:
This comment should just be updated since this function is used for anon users only. (just remove the part about the regions not being audited by the user)